src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.big {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big-field {
+  padding: 0 62px;
+  width: 100%;
+  height: 70px;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big-icon {
+  position: absolute;
+  left: 26px;
+  top: 26px;
+}
+
+.small {
+  position: relative;
+  margin-top: 20px;
+}
+
+.small-field {
+  padding: 0 33px;
+  width: 100%;
+  height: 42px;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  font-size: 14px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```

Comment:
These styles are duplicates of `.big-field` but with some small changes. Create an addition class an move there all connected styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md), the 2nd point.